src/main/java/com/minecolonies/coremod/inventory/api/CombinedItemHandler.java

```diff
@@ -272,8 +272,6 @@ public ItemStack extractItem(final int slot, final int amount, final boolean sim
         return handlers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "handlers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)